### PR TITLE
feat(status_codes) Handle 451 Unavailable For Legal Reasons

### PR DESCRIPTION
**Pull request by davidbgk**
See https://en.wikipedia.org/wiki/HTTP_451 for details.

### CODE REVIEW

**Comment by jmvrbanac** [comment]
:+1:

File: falcon/errors.py [hunk 1]
```diff
@@ -361,6 +361,25 @@ def __init__(self, title, description, retry_after=None, **kwargs):
...
+
+    See also:
+    https://datatracker.ietf.org/doc/draft-ietf-httpbis-legally-restricted-status/
+
+    Args:
+        kwargs (optional): Same as for ``HTTPError``.
+
+    """
+
+    def __init__(self, title, **kwargs):
```
Comment (kgriffs) [inline falcon/errors.py]: I noticed that you didn't mention `title` in the docstring. Did you intend for `title` to be required?

Comment (davidbgk) [inline falcon/errors.py]: At first, I was about to remove it but it generated some errors in tests so I had to put it again an I forgot to add the related docstring. Let me fix that.

Comment (kgriffs) [inline falcon/errors.py]: Got it, thanks!

**Comment by kgriffs** [comment]
@davidbgk This looks good. Would you mind rebasing on master real quick? Then I'll go ahead and merge the patch. Thanks!

**Comment by davidbgk** [comment]
@kgriffs done :)

**Comment by kgriffs** [comment]
Thanks!

**Pull request merged**